cli/args/mod.rs

```diff
@@ -244,14 +243,22 @@ impl BenchOptions {
   }
 }
 
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug)]
 pub struct FmtOptions {
   pub check: bool,
   pub options: FmtOptionsConfig,
   pub files: FilePatterns,
 }
 
 impl FmtOptions {
+  pub fn new_with_base(base: PathBuf) -> Self {
+    Self {
+      check: false,
+      options: FmtOptionsConfig::default(),
+      files: FilePatterns::new_with_base(base),
```

Comment:
See explanation for this `base` here: https://github.com/denoland/deno_config/pull/31#discussion_r1456423798